container/build.py

```diff
@@ -189,10 +215,21 @@ def cloud_build(project, container, tag, async_, package, target,
       extra_substitutions: any extra substitutions required for the given yaml file
                             mainly used for _BUCKET and _TARBALL
   '''
+
+  # We need to run clean to make sure we don't mount local build outputs
+  subprocess.call(["bazel", "clean"])
```

Comment:
should clean before both local and cloud build?

Reply:
That's where it was originally called before, but it does only seem necessary for the cloud one. Moved